### ISSUE TITLE
[low] Color Contrast: Change Password: The focus outline does not meet the contrast ratio of 3:1

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5646]
## Action Performed:
Prerequisite: the user is logged in with valid credentials.

1. Using Chrome, open https://staging.new.expensify.com/
2. Click on the user icon in the top left.
3. Click on the button "Change password", then enter the Current Password, New Password, Confirm New Password details accordingly.
5. Press Tab key to focus the button "Save".
6. Measure the contrast ratio of the blue focus outline against the green button.

## Expected Result:
The focus outline on the interactive elements should meet the minimum color contrast ratio requirements of 3:1.

## Actual Result:
The blue visible focus outline of the "Save" button (#0185FF) against the green button background (#03D47C) has a color contrast ratio of 1.84:1.

HTML:
`<div dir="auto" class="css-901oao r-lrvibr" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 13px; font-weight: 700; text-align: center;">Save</div>`

CSS:
`{
background-color: rgb(3, 212, 124);
border-radius: 8px;
border-width: 0px;
height: 40px;
opacity: 1;
padding-right: 12px;
padding-left: 12px;
}`

Other occurrences:
1. Request Money ->The visible focus outline on the "Next" button does not meet the minimum color contrast ratio requirements of 3:1 with the same color pair used.
2. Split bill ->The visible focus outline of the interactive element 'Next' button does not meet the minimum color contrast ratio requirements of 3:1 with the same color pair used.

## Workaround:
No workaround

## Area issue was found in:
Change password page, Request money overlay, Split bill overlay

## Failed WCAG checkpoints
1.4.11

## User impact:
The low-vision and color-blind users might not be able to see the focus outline of the mentioned elements.

## Suggested resolution:
Use a darker shade of blue for the outline, such as `#0061BD`, which will give a color contrast ratio of 3.10:1 against the same green button background. 

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5184304_5184304-Change_Password-The_focus_outline_does_not_meet_the_contrast_ratio](https://user-images.githubusercontent.com/88733897/136035711-2de3f565-2a48-4f46-8cea-a0644312d69a.png)
![Bug5184304_Request_Money-The_focus_outline_does_not_meet_the_contrast_ratio_other_](https://user-images.githubusercontent.com/88733897/136035725-46195269-d8d7-430f-a994-2442277ed01b.png)
![Bug5184304_Split_Bill-The_focus_outline_does_not_meet_the_contrast_ratio_other_](https://user-images.githubusercontent.com/88733897/136035738-c432ebcc-bdc7-4887-ba31-c5ce29b3a805.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
